app/src/main/AndroidManifest.xml

```diff
@@ -104,17 +104,17 @@
                 <data android:scheme="http" />
                 <data android:scheme="https" />
             </intent-filter>
-        </activity> <!-- Version < 3.0. DeX Mode and Screen Mirroring support -->
+        </activity>
         <activity
             android:name=".activity.OpmlFeedChooserActivity"
-            android:label="@string/opml_import_label" /> <!-- Version >= 3.0. DeX Dual Mode support -->
+            android:label="@string/opml_import_label" /> <!-- Version < 3.0. DeX Mode and Screen Mirroring support -->
```

Comment:
Are these changes required/related to app actions?

Reply:
Somehow it was changed automatically (together with the reference to actions.xml) when I created the actions.xml file but I'm not sure why